packages/modeling/src/operations/extrusions/extrudeHelical.js

```diff
@@ -0,0 +1,81 @@
+const { TAU } = require('../../maths/constants')
+const slice = require('./slice')
+const mat4 = require('../../maths/mat4')
+const extrudeFromSlices = require('./extrudeFromSlices')
+const geom2 = require('../../geometries/geom2')
+
+/**
+ * Perform a helical extrude of the geometry, using the given options.
+ *
+ * @param {Object} options - options for extrusion
+ * @param {Number} [options.angle=TAU] - angle of the extrusion (RADIANS) positive for right-hand rotation, negative for left-hand
```

Comment:
@Loosetooth is it possible to support 'startAngle'? both extrudeRotate and torus have this option.